code/game/machinery/OpTable_vr.dm

```diff
@@ -0,0 +1,15 @@
+/obj/machinery/optable
+	circuit = /obj/item/weapon/circuitboard/optable
```

Comment:
No construct-able optable unless we can think of a way to limit it from being used for exploration. Please remove it if you want this to be merged, otherwise it can stay as a WIP indefinitely until such a time as we have an idea about how to do that. Up to you.

Reply:
You know, you could just give the explorers a single circuit board. they still have to get all the rest from engineering. it'd be the only one of its kind.